src/Validation/Factory.php

```diff
@@ -10,6 +10,13 @@
  */
 class Factory extends BaseFactory implements FactoryContract
 {
+    /**
+     * The Validator resolver instance.
+     *
+     * @var \Closure|null
+     */
+    protected $resolver;
```

Comment:
Why was this change required?

Reply:
It was an attempt to appease PHPStan on another line - there is an `is_null` check for `$this->resolver` which PHPStan says will always be true because according to the docs for resolver, it can never be `null`. This is not strictly speaking correct, but it is how the API defines it.

It doesn't work, because now the doc is not covariant with Laravel's original docs, so I'll remove it and rely on Laravel's definition.